packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -239,6 +239,66 @@ class ToolbarSpacers extends React.Component {
 }
 ```
 
+### Sticky
+
+```js
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarContent } from '@patternfly/react-core';
+import { Button, ButtonVariant, InputGroup, TextInput, Checkbox } from '@patternfly/react-core';
```

Comment:
I see this is on the other examples here as well, but does anyone know if there's a particular reason we're splitting these imports like this?

Reply:
I looked up the history a bit and I think it's leftovers from when toolbar was experimental 
https://github.com/patternfly/patternfly-react/blob/6ec0ce4b75da8a1a4883ed61a3a2adb7dc910237/packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbar.md?plain=1#L18-L19

I combined the imports for this pr

Reply:
I imagine it happens because the IDE added the import weirdly and no one noticed in the initial PR

Reply:
@MariaAga I see the imports combined on lines 22/23 but 253/254 here are still split.